core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/colsOf.kt

```diff
@@ -262,7 +262,10 @@ public inline fun <reified C> SingleColumn<DataRow<*>>.colsOf(
  * match the given [filter] and are the given [type].
  */
 @Suppress("UNCHECKED_CAST")
-internal fun <C> ColumnsResolver<*>.colsOfInternal(type: KType, filter: ColumnFilter<C>): TransformableColumnSet<C> =
+internal inline fun <C> ColumnsResolver<*>.colsOfInternal(
```

Comment:
there are a couple more functions that be inline in this file

Reply:
Which ones?